recipes-mender/mender/files/mender.service

```diff
@@ -5,7 +5,7 @@ Description=Mender OTA update service
 Type=simple
 User=root
 Group=root
-ExecStart=/usr/bin/mender -daemon
+ExecStart=/usr/bin/mender -daemon -log-level=debug -log-file=/var/log/mender.log
```

Comment:
I'm concerned that we're adding another log file instead of just consolidating on journal/syslog. Is there anything special about mender logs that they need to be separate from the rest of the system?